examples/jsm/controls/OrbitControls.js

```diff
@@ -1179,6 +1179,8 @@ class OrbitControls extends EventDispatcher {
 
 		function onTouchStart( event ) {
 
+			if ( scope.enabled === false ) return;
```

Comment:
It seems this is redundant since there is already the same check in `onPointerDown()`. And `onTouchStart()` is called within `onPointerDown()`.

Reply:
I'll investigate - adding the check did definitely fix OrbitControls still doing something when in an AR session on a mobile phone, there may be other conditions leading to that code being called.

Reply:
OK! Can verify that this change here overlapped with a better change in the right place in our codebase :) Thank you for reviewing!